packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -1168,7 +1171,7 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
         method: this.handleOutgoingCallAlerting.name,
       });
     } catch (err) {
-      log.warn('Call Progress failed with Mobius', {
+      log.warn('Failed to progress the call', {
```

Comment:
This can be reworded to "Failed to signal call progression"

Reply:
Changed.